proj/src/main/webapp/static/signup.html

```diff
@@ -13,6 +13,9 @@
     <!-- Font  -->
     <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap" rel="stylesheet">
 
+    <!-- Google api key -->
+    <script async defer src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCtEo_jPb_ThnU_HTaYh4U_t-8MLZ9ulBo">
```

Comment:
You might want to change this to use the other key so we can delete this key and we won't have redundant keys.